tests/unit/components/app-text-area/AppTextArea.spec.jsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react'
+import { vi } from 'vitest'
+import AppTextArea from '~/components/app-text-area/AppTextArea'
+
+vi.mock('~/components/app-text-field/AppTextField', () => ({
+  __esModule: true,
+  default: function () {
+    return <div>AppTextField</div>
+  }
+}))
+
+vi.mock('@mui/material', async () => {
+  const actual = await vi.importActual('@mui/material')
+  return {
+    ...actual,
+    Typography: vi.fn(({ children }) => <p>{children}</p>)
```

Comment:
According to the component logic `Typography` should obtain prop `color`. 
It allow you cover 33th line of code to provide different value in `AppTextArea`.

Reply:
done! 
![img-app-text-area-unit-test-coverage](https://github.com/Space2Study-UA-1125/frontEnd/assets/127215334/14ad6735-f8ed-4f37-b90f-d36a32e8bc6e)